en/conf.py

```diff
@@ -44,7 +44,7 @@
 
 # General information about the project.
 project = u'CUBRID'
-copyright = u'2017, Search Solution Corporation'
+copyright = u'2021, CUBRID Corporation'
```

Comment:
```suggestion
copyright = u'2016, CUBRID Corporation'
```
Is the year of copyright right?